docs/blog/posts/introduction-to-bpfd.md

```diff
@@ -0,0 +1,611 @@
+cert-manager         cert-manager-webhook-545bd5d7d8-z5brw             1/1     Running   0          57s
+kube-system          coredns-565d847f94-gjjft                          1/1     Running   0          61s
+kube-system          coredns-565d847f94-mf2cq                          1/1     Running   0          61s
+kube-system          etcd-test-bpfd-control-plane                      1/1     Running   0          76s
+kube-system          kindnet-lv6f9                                     1/1     Running   0          61s
+kube-system          kube-apiserver-test-bpfd-control-plane            1/1     Running   0          76s
+kube-system          kube-controller-manager-test-bpfd-control-plane   1/1     Running   0          77s
+kube-system          kube-proxy-dtmvb                                  1/1     Running   0          61s
+kube-system          kube-scheduler-test-bpfd-control-plane            1/1     Running   0          78s
+local-path-storage   local-path-provisioner-684f458cdd-8gxxv           1/1     Running   0          61s
+```
+
+Note that we have the bpfd-operator, bpf-daemon and cert-manager pods running.
+
+6\. Deploy the XDP counter program and user space application
+
+```bash
+kubectl apply -f https://github.com/bpfd-dev/bpfd/releases/download/v0.2.1/go-xdp-counter-install-v0.2.1.yaml
+```
+
+7\. Confirm that the programs are loaded
+
+Userspace program:
+
+```bash
+kubectl get pods -n go-xdp-counter
+```
+
+```bash
+NAME                      READY   STATUS              RESTARTS   AGE
+go-xdp-counter-ds-9lpgp   0/1     ContainerCreating   0          5s
+```
+
+XDP program:
+
+```bash
+kubectl get xdpprograms.bpfd.io -o wide
+```
+
+```bash
+NAME                     SECTIONNAME   NODESELECTOR   PRIORITY   INTERFACESELECTOR               PROCEEDON
+go-xdp-counter-example   stats         {}             55         {"primarynodeinterface":true}   ["pass","dispatcher_return"]
+```
+
+8\. Confirm that the counter program is counting packets.
+
+Notes:
+
+- The counters are updated every 5 seconds, and stats are being collected
+for the pod's primary node interface, which may not have a lot of traffic.
+However, running the `kubectl` command below generates traffic on that
+interface, so run the command a few times and give it a few seconds in between
+to confirm whether the counters are incrementing. 
+- Replace "go-xdp-counter-ds-9lpgp" with the go-xdp-counter pod name for
+your deployment.
+
+```bash
+kubectl logs go-xdp-counter-ds-9lpgp -n go-xdp-counter | tail
+```
+
+```bash
+2023/09/05 16:58:21 1204 packets received
+2023/09/05 16:58:21 13741238 bytes received
+
+2023/09/05 16:58:24 1220 packets received
+2023/09/05 16:58:24 13744258 bytes received
+
+2023/09/05 16:58:27 1253 packets received
+2023/09/05 16:58:27 13750364 bytes received
+```
+
+9\. Deploy the `xdp-pass-all-nodes` program with `priority` set to 50 and
+  `proceedon` set to `drop` as shown below
+
+```bash
+kubectl apply -f - <<EOF
+```
```

Comment:
In this sample you put a space between the `kubectl apply` command and the associated yaml, but in the followup sample you do not. I think I like the second output, but they should be consistent.

Reply:
I'll change it.  I was trying to see if it formatted the yaml block differently, but it didn't.  I'll make it like the next one.